crates/bevy_ecs/src/query/mod.rs

```diff
@@ -21,6 +21,12 @@ pub use par_iter::*;
 pub use state::*;
 pub use world_query::*;
 
+use crate as bevy_ecs;
```

Comment:
What does this line do? It is related to the derive macro?

Reply:
For the derive macro, yea